qemu/tests/cfg/virtio_fs_migration_on_error.cfg

```diff
@@ -0,0 +1,92 @@
+- virtio_fs_migration_on_error:
+    no RHEL.6 RHEL.7 RHEL.8.0 RHEL.8.1
+    no Win2008 Win7 Win2012 Win2012..r2 Win8 Win8.1
+
+    type = virtio_fs_migration_on_error
+    virt_test_type = qemu
+    required_qemu = [9.0.0,)
+    kill_vm = yes
+    start_vm = yes
+
+    filesystems = fs
+    filesystems_migration = targetfs
+    fs_driver = virtio-fs
+    fs_source_type = mount
+    fs_source_dir_fs = /var/tmp/virtio_fs_source
+    fs_source_dir_targetfs = /var/tmp/virtio_fs_target
+    fs_target = myfs
+    fs_dest = /mnt/${fs_target}
+    force_create_fs_source = no
+    remove_fs_source = no
+    fs_driver_props = {"queue-size": 1024}
+    driver_name = viofs
+
+    test_file = test_file
+    test_data = hello_virtiofs
+    pre_command = mkdir -p ${fs_source_dir_fs} ${fs_source_dir_targetfs}
+    post_command = rm -rf ${fs_source_dir_fs} ${fs_source_dir_targetfs}
+    read_file_cmd = "cat %s/${test_file}"
+    fs_binary_extra_options = " --cache auto"
+    on_error = abort
+
+    share_mem = yes
+    vm_mem_share = yes
+    # Note that memory-backend-file can cause error when doing live migration
+    vm_mem_backend = memory-backend-memfd
```

Comment:
@zhencliu I think the cfg still use memory-backend-memfd instead of memory-backend-file, btw I didn't find this case in our "KVM QE Feature Cases Tracker" maybe we could change the default to memory-backend-file?